src/analysis/calculus/cont_diff.lean

```diff
@@ -2602,3 +2617,53 @@ begin
 end
 
 end
+
+section apply
+
+lemma norm_iterated_fderiv_within_clm_apply {f : E → (F →L[𝕜] G)} {g : E → F} {s : set E} {x : E}
+  {N : ℕ∞} {n : ℕ} (hf : cont_diff_on 𝕜 N f s) (hg : cont_diff_on 𝕜 N g s) (hs : unique_diff_on 𝕜 s)
+  (hx : x ∈ s) (hn : ↑n ≤ N) :
+    ‖iterated_fderiv_within 𝕜 n (λ y, (f y) (g y)) s x‖ ≤
+    (finset.range (n + 1)).sum (λ i, ↑(n.choose i) * ‖iterated_fderiv_within 𝕜 i f s x‖ *
+      ‖iterated_fderiv_within 𝕜 (n - i) g s x‖) :=
+begin
+  let B : (F →L[𝕜] G) →L[𝕜] F →L[𝕜] G :=
+  continuous_linear_map.flip (continuous_linear_map.apply 𝕜 G),
+  have hB : ‖B‖ ≤ 1 :=
+  begin
+    simp only [continuous_linear_map.op_norm_flip, continuous_linear_map.apply],
+    refine continuous_linear_map.op_norm_le_bound _ zero_le_one (λ f, _),
+    simp only [continuous_linear_map.coe_id', id.def, one_mul],
+  end,
+  refine (B.norm_iterated_fderiv_within_le_of_bilinear hf hg hs hx hn).trans _,
```

Comment:
can you use instead `continuous_linear_map.norm_iterated_fderiv_within_le_of_bilinear_of_le_one`?